### PR TITLE
Test useMasterDetails

### PR DESCRIPTION
**Pull request by ErickGarciaRios**
Test useMasterDetails.ts

### CODE REVIEW

File: test/components/masterDetailsComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { useMasterDetails } from '../../src/useMasterDetails';
+
+export const MasterDetailsComponent = () => {
+    const [open, openDetails] = useMasterDetails();
+    return (
+        <>
+            <div>
+                <span data-testid='openValue'>{ open ? 'Open' : 'Closed'}</span>
+                <button onClick={()=>openDetails()}>Open/Close</button>
```
Comment (geoperez) [inline test/components/masterDetailsComponent.tsx:10]: Can use openDetails without the lambda?

Comment (ErickGarciaRios) [reply test/components/masterDetailsComponent.tsx]: done

File: test/components/masterDetailsComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { useMasterDetails } from '../../src/useMasterDetails';
+
+export const MasterDetailsComponent = () => {
+    const [open, openDetails] = useMasterDetails();
+    return (
+        <>
```
Comment (geoperez) [inline test/components/masterDetailsComponent.tsx:7]: You dont need a wrapper if you have div. Keep fragment, it's better

Comment (ErickGarciaRios) [reply test/components/masterDetailsComponent.tsx]: done

**Pull request merged**